source/libs/features.tsx

```diff
@@ -12,34 +12,35 @@ type callerFunction = (callback: VoidFunction) => void;
 type featureFunction = () => boolean | void;
 type featurePromisedFunction = () => Promise<boolean | void>;
 
-interface featureShortcuts {
-	[key: string]: string
+interface FeatureShortcuts {
+	[key: string]: string;
 }
 
 interface Shortcut {
-	hotkey: string,
-	description: string,
+	hotkey: string;
+	description: string;
 }
 
 interface GlobalOptions {
-	disabledFeatures: string,
-	customCSS: string,
-	logging: boolean,
-	log?: (...args: any[]) => void
+	disabledFeatures: string;
+	customCSS: string;
+	logging: boolean;
+	log?: (...args: unknown[]) => void;
 }
 
 interface FeatureDetails {
-	id: string,
-	include?: BooleanFunction[],
-	exclude?: BooleanFunction[],
-	init: featureFunction | featurePromisedFunction,
-	deinit?: Function,
-	load?: callerFunction | Promise<void>,
-	shortcuts?: featureShortcuts,
+	id: string;
+	include?: BooleanFunction[];
+	exclude?: BooleanFunction[];
+	init: featureFunction | featurePromisedFunction;
+	// TODO: Remove the disable comment
+	deinit?: Function; // eslint-disable-line @typescript-eslint/ban-types
```

Comment:
Why `@typescript-eslint/ban-types`? Is `Function` not normally allowed? Should this just be `function`?

Reply:
Ah right

Reply:
Done: https://github.com/sindresorhus/refined-github/pull/1740/commits/15681c02bd85a8b2efa02f88030edd67ebf2b938